go-controller/pkg/testing/testing.go

```diff
@@ -1,10 +1,18 @@
 package testing
 
 import (
+	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega/format"
 	kruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"os"
 )
 
+func OnSupportedPlatformsIt(description string, f interface{}) {
+	if os.Getenv("NOROOT") != "TRUE" {
+		ginkgo.It(description, f)
+	}
```

Comment:
Do we want to log a message that says, the test is being skipped?

Reply:
let me see if i can make it do a ginkgo.Skip